packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs

```diff
@@ -30,8 +30,8 @@
     <InstallExecuteSequence>
       <InstallValidate Suppress="yes">FAKE_PROPERTY</InstallValidate>
       <Custom Action="StopUpdater" Before="InstallValidate"/>
+      <Custom Action="StopMainApp" Before="StopGUI"/>
```

Comment:
There's no scenario in which the GUI will try to restart the main app, is there?